native/src/components/HeaderContainer.ts

```diff
@@ -47,6 +47,18 @@ const mapStateToProps = (state: StateType, ownProps: OwnPropsType): StatePropsTy
           route: routeName as typeof simpleRoutes extends (infer U)[] ? U : never
         })
       : null
+  const newsRouteShareUrl =
+    ownProps.scene.route.name === NEWS_ROUTE
+      ? urlFromRouteInformation({
+          cityCode,
+          languageCode,
+          route: NEWS_ROUTE,
+          // @ts-ignore
+          newsType: ownProps.scene.route.params.newsType,
+          // @ts-ignore
+          newsId: ownProps.scene.route.params.newsId
+        })
```

Comment:
sharing did not work for me for tunews content. For me it created this link: https://integreat.app/testumgebung/de/news/local

Reply:
Done in #524